CvLocalization/include/CvLocalization.h

```diff
@@ -288,6 +288,9 @@ namespace Localization
 		CvLocalizationAPI String(const char* szString);
 		CvLocalizationAPI String(const char* szString, unsigned int uiLen);
 		CvLocalizationAPI String(const String& other);
+#ifdef __clang__
+		inline explicit String(const CvString& string) : String(string.data(), string.length()) {}
+#endif // __clang__
```

Comment:
are you sure this is necessary?

Reply:
Not certain. This was just the easiest way for me to get it building right away. The ifdef is necessary because otherwise the VS2008 compiler errors because of the constructor delegation. If you've got something prettier I'd be happy to change it (it's ugly).

Reply:
I'm fairly sure that you can just delete this constructor. https://github.com/Hello71/Community-Patch-DLL/commit/6837bcb8d10a6718412e438c4ab7661f7a2e67ab is all that I needed. The way that this works on MSVC is that `Localization::String::String(const char *)` exists, so `CvString::operator const char *()` is inferred. That is an MSVC extension though, since it's inferring both the `CvString` -> `const char *` and `const char *` -> `Localization::String` conversions. This can be fixed by adding a single implicit conversion from `CvString` to `Localization::String` or by constructing the `Localization::String` directly with a `CvString` so that the second conversion doesn't need to be inferred.

Reply:
You're completely right about it being unnecessary. I'm not sure how I managed to miss that.